src/components/menus/ViewMenu.svelte

```diff
@@ -2,50 +2,75 @@
 
 <script lang="ts">
```

Comment:
I probably missed various conversations but is the plan to hold off on the different tabs?
<img width="344" alt="image" src="https://user-images.githubusercontent.com/4419607/216724654-2f787be0-c14e-4a23-8a81-5e7076741e4a.png">

Reply:
Doh! My brain ignored this because I saw the "Mission Model" tab and thought it could ignore all the tabs. You're right, this should have the "All"/"My Views" tabs, but not the "Mission Model" tab. I can add that in quickly tomorrow.